app.py

```diff
@@ -0,0 +1,25 @@
+from dash import Dash, dcc, html, Input, Output
+import os
+
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
+
+app = Dash(__name__, external_stylesheets=external_stylesheets)
+
+server = app.server
```

Comment:
Do we use `server` anywhere?  I don't see any usages.  If it isn't used, can we delete it?